recipe/meta.yaml

```diff
@@ -103,8 +108,11 @@ outputs:
         - zlib
       run:                                                            # [not win]
         - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}     # [not win]
-      run_constrained:                                                # [not win]
-        - llvmdev   {{ version }}                                     # [not win]
```

Comment:
This part I don't understand either. The run-constraint _from_ `llvm` _on_ `llvmdev` should have been there already, but it still shows up in the diff of #141.